integrations/linear/src/handler.ts

```diff
@@ -21,16 +22,12 @@ export const handler: bp.IntegrationProps['handler'] = async ({ req, ctx, client
   const linearEvent: LinearEvent = JSON.parse(req.body)
   linearEvent.type = linearEvent.type.toLowerCase() as LinearEvent['type']
 
-  const webhookSignatureHeader = req.headers[LINEAR_WEBHOOK_SIGNATURE_HEADER]
-  if (!webhookSignatureHeader) {
-    return
+  if (!_isWebhookProperlyAuthenticated({ req, linearEvent, ctx })) {
+    throw new RuntimeError(
```

Comment:
**note:** there's no need to use a `RuntimeError` here since the error response is sent to the webhook caller, not the bot

Reply:
alright, I'll change that